pest/src/iterators/tokens.rs

```diff
@@ -92,6 +92,12 @@ impl<'i, R: RuleType> Tokens<'i, R> {
     }
 }
 
+impl<'i, R: RuleType> ExactSizeIterator for Tokens<'i, R> {
+    fn len(&self) -> usize {
+        self.end - self.start
```

Comment:
likely ok, but perhaps good to check / add a test that this works with non-ASCII UTF-8 characters